src/imgix-api.js

```diff
@@ -23,27 +26,25 @@
 
     // default ImgixAPI settings passed in during instantiation
     const DEFAULTS = {
-        version: 1
+        version: 1,
+        apiKey: null
     };
 
     const ImgixAPI = (() => {
         function ImgixAPI(opts = {}) {
-            this.settings = {};
-
-            if (opts.version) {
-                this.settings.version = opts.version;
-            } else {
-                this.settings = DEFAULTS;
-            }
+            this.settings = Object.assign({}, DEFAULTS, opts);
```

Comment:
Can we throw an error here if no API key is provided? None of these requests will work if no key is provided.